README reworked (#24)
otakuhqz/Omega: README.md

    </tr>
</table>

## Donations <br/>
[![Donate on Liberapay](https://liberapay.com/assets/widgets/donate.svg)](https://liberapay.com/otakuhqz/donate)<br/>
<a href="https://www.buymeacoffee.com/2ulp4f5" target="_blank">
    <img src="https://cdn.buymeacoffee.com/buttons/v2/default-blue.png" 
    alt="Buy Me A Coffee" width="150" >
</a><br/>
[![Donate on PayPal](https://img.shields.io/badge/PayPal-Donate%20Now-red.svg?style=for-the-badge)](https://paypal.me/saulhenriquez)

## Contributions

> If you have knowledge in Java or Kotlin you can work on additional features!

Author: ```suggestion
If you know Java or Kotlin you can work on additional features!
```

> I will be happy to work on features you would like to include, just open an issue.

Author: ```suggestion
I will happily work on features you would like to have, just open an issue.
```
